### PR TITLE
Add gravatar pictures for regular users and associated specs

### PR DESCRIPTION
**Pull request by scarfdestroyer**

### CODE REVIEW

**Comment by scarfdestroyer** [comment]
Potentially satisfying issue https://github.com/code-corps/code-corps-api/issues/83

File: spec/workers/add_profile_picture_from_gravatar_worker_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe AddProfilePictureFromGravatarWorker do
+  before do
+    @user = create(:user, email: "fake_email@email.com")
+    @user_gravatar = create(:user, email: "bradyrudesill@gmail.com")
+  end
+
+  context "when the user does not have a gravatar", vcr: { cassette_name: 'workers/add gravatar image worker/adds gravatar' } do
+    it "a photo has not been added" do
```
Comment (joshsmith) [inline spec/workers/add_profile_picture_from_gravatar_worker_spec.rb]: These `it` blocks need re-written to read well.

**Comment by joshsmith** [comment]
That change should be all that's needed and then :shipit: :+1:

**Comment by joshsmith** [comment]
Your its don't read the same. "adds the photo"/"does not add the photo" context-switches less.

**Comment by joshsmith** [comment]
@scarfdestroyer update the specs when you get a moment and this will get merged.

**Pull request merged**